docroot/themes/contrib/civictheme/civictheme_library/components/01-atoms/button/button.twig

```diff
@@ -25,6 +26,7 @@
 {% set type = type in ['primary', 'secondary', 'tertiary'] ? type : null %}
 {% set size = size in ['large', 'regular', 'small'] ? size : 'regular' %}
 {% set icon_placement = icon_placement in ['before', 'after'] ? icon_placement : 'after' %}
+{% set icon_size = icon_size in ['large', 'regular', 'small'] ? icon_size : null %}
```

Comment:
@joshua-salsadigital 
I have deliberately removed this functionality to make sure that icons within buttons follow button sizes and do not have any custom sizes.

Please remove these and refactor the code to only use the button size.

Reply:
Reverted the change, Added size as per button only